src/main/java/uk/gov/pay/webhooks/app/WebhooksApp.java

```diff
@@ -79,4 +85,23 @@ public DataSourceFactory getDataSourceFactory(WebhooksConfig configuration) {
         bootstrap.getObjectMapper().getSubtypeResolver().registerSubtypes(GovUkPayDropwizardRequestJsonLogLayoutFactory.class);
     }
 
+    private void initialiseMetrics(WebhooksConfig configuration, Environment environment) {
+        DatabaseMetricsService metricsService = new DatabaseMetricsService(configuration.getDataSourceFactory(), environment.metrics(), "webhooks");
+
+        environment
+                .lifecycle()
+                .scheduledExecutorService("metricscollector")
+                .threads(1)
+                .build()
+                .scheduleAtFixedRate(metricsService::updateMetricData, 0, GRAPHITE_SENDING_PERIOD_SECONDS / 2, TimeUnit.SECONDS);
+
+        GraphiteSender graphiteUDP = new GraphiteUDP(configuration.getGraphiteHost(), Integer.parseInt(configuration.getGraphitePort()));
+        GraphiteReporter.forRegistry(environment.metrics())
+                .prefixedWith(SERVICE_METRICS_NODE)
+                .convertRatesTo(TimeUnit.MINUTES)
+                .convertDurationsTo(TimeUnit.MILLISECONDS)
+                .build(graphiteUDP)
+                .start(GRAPHITE_SENDING_PERIOD_SECONDS, TimeUnit.SECONDS);
+    }
```

Comment:
This looks reassuringly similar to what we have in connector.

Reply:
Yes, I took most of the code from there 😆